lua/neogit/config.lua

```diff
@@ -69,9 +69,14 @@ end
 ---@class NeogitConfigPopup Popup window options
 ---@field kind WindowKind The type of window that should be opened
 
+---@alias StagedDiffSplitKind
+---| "split" Open in a split
+---| "vsplit" Open in a vertical split
```

Comment:
This can also include `split_above` and `auto`. `tab` and `floating` would _work_, but I think the result would be poor.

Reply:
Makes sense. Pushed.

Reply:
Thanks!